playbooks/roles/airship-deploy-tempest/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+# tasks file for airship-deploy-tempest
+- name: Load standard variables
+  include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+  tags:
+    - always
+
+- name: Set site path
+  set_fact:
+    site_path: "{{ upstream_repos_clone_folder }}/airship/treasuremap/site/{{ socok8s_site_name }}"
+  tags:
+    - always
+
+- name: Ensure site directory for {{ socok8s_site_name }} exists
+  become: yes
+  file:
+    path: "{{ site_path }}"
+    state: directory
+
+- name: Ensure tempest chart directory exists
+  become: yes
+  file:
+    path: "{{ site_path }}/software/charts/osh/openstack-tempest"
+    state: directory
+
+- name: Determine tempest test type (smoke or all)
+  set_fact:
+    test_args: "{% if tempest_test_type == 'all' %}--regex 'tempest.api.*'{% else %}--smoke{% endif %}"
```

Comment:
This appears to be needed during `TASK [airship-deploy-ucp : Create site software manifests]`, is there somewhere we can put this so that the `airship-deploy-ucp` role can access it as well?

Reply:
Hmm...it shouldn't be. The entire 'values' section of the tempest template should get ignored during airship-deploy-ucp. Is 'deploy_tempest: true' set anywhere in your env? I was able to pass that stage during my 3 test runs.